packages/components/src/Chip/Chip.types.d.tsx

```diff
@@ -1,11 +1,20 @@
 import React, { PropsWithChildren } from "react";
 
 export interface ChipProps extends PropsWithChildren {
+  /**
+   * If you need Chip to be part of a form element, this makes it not a button, but just a div so you can wrap it in a label.
+   */
+  readonly actAsFormElement?: boolean;
   /**
    * Accessible label, which can be different from the primary label.
    */
   readonly ariaLabel?: string;
 
+  /**
+   * Enables pass through of a data-testid. This is legacy functionality to allow old tests to pass.
```

Comment:
> This is legacy functionality to allow old tests to pass

Would we call this legacy? We still use testID's in new tests to make sure we're rendering the right component.

What do you think about a description like this: `Used to locate this view in end-to-end tests`

Reply:
I had initially stripped out the data-testid because I didn't think we needed it, we could just grab the element via the label (since we're setting it) or the role, but a lot of the existing tests were using the data-testid so I put it back in with that comment so the tests I didn't were needed (they were) would pass.

You're right though, the comment about legacy functionality was for us and not consumers. I included a modified version of your comment in my own to reflect the true nature of the prop. Thanks Taylor!

Reply:
Perfect - love the update!